### DISCUSSION TITLE
Windows系统clone代码之后，报错invalid path

### DISCUSSION THREAD

**Discussion by HawtinZeng** [antvis/G6#6230] [Show and tell]
![image](https://github.com/user-attachments/assets/5b690168-ac15-4383-9763-4643339b6900)

原因：Windows系统上git NTFS filesystem默认为true，因此检测到packages/g6/__tests__/perf-report/9821ed36_2024-08-19_16:33:46.json这个文件中的冒号不合法。导致git不能切换到v5分支。仓库内文件不显示。

**Comment by HawtinZeng** [comment]
解决方法：https://dev.to/jumbo02/how-to-fix-error-invalid-path-when-working-with-git-on-windows-16kf

**Comment by Aarebecca** [comment]
这边会尽快修复文件名在某些系统下不合法的问题
